arch_diagrams/create_diagrams.py

```diff
@@ -0,0 +1,81 @@
+from diagrams import Cluster, Diagram, Edge
+from diagrams.onprem.vcs import Github
+from diagrams.onprem.ci import GithubActions, TravisCI, DroneCI, CircleCI
+from diagrams.azure.devops import Pipelines
+from diagrams.generic.storage import Storage
+from diagrams.generic.place import Datacenter
+from diagrams.generic.compute import Rack as Heroku
+
+with Diagram(name="Conda-Forge Conceptual", show=False, graph_attr=dict(concentrate='True')):
+    staged_recipes = Github(label='staged-recipes')
+    with Cluster('Build CIs'):
+        cis = [Pipelines(label='Windows, Linux, \nMacOS'), DroneCI(label='aarch64'), TravisCI(label='powerpc'),
+               CircleCI('misc')]
+
+    linter = Heroku(label='metadata linter')
+    rerender = GithubActions(label='feedstock rerender')
+    create_feedstock = GithubActions(label='create feedstock repo')
+    autotick = GithubActions(label='autotick bot')
+
+    new_feedstock = Github(label='new feedstock')
+    anaconda_server = Storage(label='Anaconda Servers')
+    pinnings = Github(label='ABI pins')
+
+    internet = Datacenter(label='new version numbers')
+    validation = Heroku(label='pkg validation')
+
+    admin_migrations = CircleCI('admin migrations')
+
+
+    staged_recipes >> cis >> staged_recipes
+    staged_recipes >> linter >> staged_recipes
+    staged_recipes >> create_feedstock >> new_feedstock >> cis >> validation >> anaconda_server
```

Comment:
I'm not sure if package validation needed for the conceptual diagram as I see it as more of an implementation detail. Perhaps "Anaconda Servers" should be "conda channel"?

Reply:
We maintain the validation with infrastructure we control so it needs to be here.

Reply:
I agree it needs to be the "Conda-Forge Implementation" diagram but I'm not sure it needs to be in the "Conda-Forge Conceptual" diagram though I don't feel that strongly about it if others disagree.

Reply:
ah sure agreed!

Reply:
Hmm, validation might be an independent concept, since it isn't formally owned by any of the other pieces. Also the fact that we validate our packages is important to the running of conda-forge, since is a barrier to some potential attacks. I would call the implementation detail that this is run off of the heroku server for instance. I'm happy to be turned around on this though.